bcp47/library/Data/BCP47/Internal/LanguageExtension.hs

```diff
@@ -27,6 +33,7 @@ instance Arbitrary LanguageExtension where
     components <- replicateM 3 $ alphaString 3
     pure . LanguageExtension $ pack $ intercalate "-" components
 
+-- | Parse a 'LanguageExtension' subtag from 'Text'
```

Comment:
Do you want `@` tags?

Reply:
I never know what the right things is here. `'` makes `haddock` link to the document for that symbol. Where as `@` just formats it monospaced. I think `'` also mono spaces. This screenshot would elude to that.

![image](https://user-images.githubusercontent.com/545655/58906897-9c986d00-86d2-11e9-8fd8-15f8d8cc49e0.png)

Reply:
I think I do `@'Whatever'@`, but only because @pbrisbin always calls it out in code review :stuck_out_tongue:

Reply:
My understanding was:

```
- 'Linked'
- @Monospace@
- @'LinkedMonospace'@, but
- '@AlsoLinkedMonospace@'
```

i.e. nesting doesn't matter, it's like `<i><b>foo</b></i>` vs `<b><i>foo</i></b>`.

I could be wrong on that last part, I've never tested that AWFUL `'@...@'` style :)

From [the docs](https://www.haskell.org/haddock/doc/html/ch03s08.html):

> Monospaced (or typewriter) text is indicated by surrounding it with @...@. Other markup is valid inside a monospaced span: for example @'f' a b@ will hyperlink the identifier f inside the code fragment.

> I think `'` also mono spaces. This screenshot would elude to that.

That I did not know, and I can't find anything in the docs saying it should work like that. I wonder if it depends on Haddock version? It certainly seems reasonable.

Personally, I would prefer to keep doing `@'...'@` if only to avoid throwing tons of existing docs into the abyss of inconsistency :man_shrugging: